workbench/icons/Makefile.am

```diff
@@ -0,0 +1,20 @@
+icondir = $(datadir)/icons/hicolor/16x16/apps
```

Comment:
> Makes sense, why would you want to distribute files that you've indicated need to be installed? </sarcasm>

Thats autofools for ya :stuck_out_tongue_winking_eye: , but it is documented behaviour for `_DATA`, installs but not dist unless its explicit.